### PR TITLE
Use Deno EventEmitter polyfill

### PR DESCRIPTION
**Pull request by arnohovhannisyan**
This fixes importing CAC in Deno since the `events` module doesn't exist there.

### CODE REVIEW

**Comment by egoist** [comment]
don't edit `mod.js` it's auto-generated, we should update rollup to inline `events` module: https://github.com/cacjs/cac/blob/master/rollup.config.js

**Comment by arnohovhannisyan** [comment]
@egoist I have updated the rollup config, now it replaces `events` with the Deno URL automatically.

**Comment by egoist** [comment]
Can you run rollup locally and commit the changes to `mod.js` as well? 😄

File: rollup.config.js [hunk 1]
```diff
@@ -1,3 +1,27 @@
...
+      if (name === 'events') {
+        return name
+      }
+      return null
+    },
+    load(id) {
+      if (id === 'events') {
+        return [
+          'export { default } from "https://deno.land/std/node/events.ts"',
+          'export * from "https://deno.land/std/node/events.ts"'
```
Comment (egoist) [inline rollup.config.js]: I guess `export *` will export `default` as well

**Pull request merged**